src/sky.cpp

```diff
@@ -564,30 +715,53 @@ void Sky::update(float time_of_day, float time_brightness,
 	m_sunlight_seen = sunlight_seen;
 	m_bodies_visible = true;
 
-	bool is_dawn = (time_brightness >= 0.20 && time_brightness < 0.35);
+	bool is_dawn = (time_brightness >= 0.10 && time_brightness < 0.35);
 
 	/*
 	Development colours
 
-	video::SColorf bgcolor_bright_normal_f(170. / 255, 200. / 255, 230. / 255, 1.0);
-	video::SColorf bgcolor_bright_dawn_f(0.666, 200. / 255 * 0.7, 230. / 255 * 0.5, 1.0);
-	video::SColorf bgcolor_bright_dawn_f(0.666, 0.549, 0.220, 1.0);
-	video::SColorf bgcolor_bright_dawn_f(0.666 * 1.2, 0.549 * 1.0, 0.220 * 1.0, 1.0);
-	video::SColorf bgcolor_bright_dawn_f(0.666 * 1.2, 0.549 * 1.0, 0.220 * 1.2, 1.0);
+	video::SColorf sky_bottom_bright_normal_f(170. / 255, 200. / 255, 230. / 255, 1.0);
+	video::SColorf sky_bottom_bright_dawn_f(0.666, 200. / 255 * 0.7, 230. / 255 * 0.5, 1.0);
+	video::SColorf sky_bottom_bright_dawn_f(0.666, 0.549, 0.220, 1.0);
+	video::SColorf sky_bottom_bright_dawn_f(0.666 * 1.2, 0.549 * 1.0, 0.220 * 1.0, 1.0);
+	video::SColorf sky_bottom_bright_dawn_f(0.666 * 1.2, 0.549 * 1.0, 0.220 * 1.2, 1.0);
```

Comment:
Nuuuuu, please don't change these variable names, it's completely unnecessary to do so.

Reply:
Actually, since in MT master we have
`video::SColor cloudyfogcolor = m_bgcolor;`
It seems clearest to change `bgcolor` to `cloudyfogcolor` everywhere. I agree with you that `bgcolor` is unclear and confusing. But please keep `skycolor` unchanged as it is accurate and familiar. So then the names are improved but changes are minimised.